src/xrEngine/MonitorManager.cpp

```diff
@@ -27,8 +27,15 @@ MonitorsManager::ResolutionPair MonitorsManager::GetMinimalResolution()
 MonitorsManager::ResolutionPair MonitorsManager::GetMaximalResolution()
 {
     const ResolutionsMap& resolutions = Monitors[Vid_SelectedMonitor];
-    const auto it = resolutions.crbegin();
-    return it->first;
+    if(!resolutions.empty())
+    {
+        const auto it = resolutions.crbegin();
+        return it->first;
+    }
+    else
+    {
+        return ResolutionPair(640,480);
+    }
```

Comment:
Well... This was wrong decision to change the order from one to zero.

Let's return it back then?

Reply:
Ok, let`s do it. Close this PR then?